src/parser/mod.rs

```diff
@@ -6697,6 +6697,8 @@ impl<'a> Parser<'a> {
             None
         };
 
+        let ignore = dialect_of!(self is MySqlDialect) && self.parse_keyword(Keyword::IGNORE);
```

Comment:
Can you please also add support for GenericDialect?

Reply:
Generic support added